### PR TITLE
DTFS2-4380 - central api bank endpoints, TFM mapping bank in GEF deal

### PR DESCRIPTION
**Pull request by ttbarnes**
- central API get & create bank
- fetch and return bank by id when getting a GEF deal
- map bank name in UI graphql reducer

### CODE REVIEW

File: dtfs-central-api/src/v1/controllers/portal/deal/get-deal.controller.js [hunk 1]
```diff
@@ -95,10 +96,15 @@ const findOneGefDeal = async (_id, callback) => {
   if (deal) {
     deal.facilities = await findAllGefFacilitiesByDealId(_id);
```
Comment (eddieleeper) [inline dtfs-central-api/src/v1/controllers/portal/deal/get-deal.controller.js:97]: Low priority - These calls could be done in parallel

**Review by eddieleeper — APPROVED** [review]
Just an observation that the apis in this service don't have any authentication requirements.

File: dtfs-central-api/src/v1/controllers/portal/deal/get-deal.controller.js (hunk 1 above)
Comment (ttbarnes) [reply dtfs-central-api/src/v1/controllers/portal/deal/get-deal.controller.js:97]: Good point, they'll be another call we need to add here so i'll do this in another PR

**Comment by ttbarnes** [comment]
RE Authentication - I suspect the central API might be locked down in Azure @pricea1 ?

**Pull request merged**